### ISSUE TITLE
can't use with webpack

### ISSUE THREAD

**Issue by johndeighan** [substack/static-eval#17]
I've created a module that uses static-eval. In node.js, it's working great. When I try to use webpack to create a bundle including my module, esprima and static-eval, I get this error:

 require function is used in a way in which dependencies cannot be statically extracted

If I comment out references to static-eval, webpack runs fine, so it looks like the problem is in static-eval. I'm using webpack 2.5.1. Do you know of a solution?

**Comment by substack** [comment]
This module doesn't do anything fancy. Perhaps the problem is upstream in escodegen? Looking through the source of that module, the `require()` usage also looks pretty ordinary.

**Comment by johndeighan** [comment]
In my case, I only wanted a parser to handle evaluating expressions. So, I was able to comment out this line:

var unparse = require('escodegen').generate;

then in the code that used unparse() - which handled FunctionExpressions, which I did not want to allow anyway, I commented out that code, too, and threw an exception. 

After removing references to escodegen that way, webpack bundled things up nicely without error. I think that strongly suggests that you are right. Now, my problem is solved. Since the issue doesn't seem to be with static-eval, I'm going to close the issue.
